### ISSUE TITLE
The <label> element does not have descriptive text that identifies the expected input (Accessibility)

### ISSUE THREAD

**Issue by Bsujeet** [IBM/carbon-components-angular#1535]
## Detailed description

> Describe in detail the issue you're having.

In my project (MCMP) i don't need  "onText" or "offText" or "lable" for <ibm-toggle>.
I am using <ibm-toggle> component.
This is Accessibility testing issue.
The <label> element does not have descriptive text that identifies the expected input

> Is this a feature request (new component, new icon), a bug, or a general issue?

Accessibility Bug

> Is this issue related to a specific component?

ibm-toggle

> What did you expect to happen? What happened instead? What would you like to see changed?

The component should not give Accessibility defect if i don't want offtext or ontext

> What browser are you working in?

Chrome

> Tools

IBM Equal Access Accessibility Checker

> What version of the Carbon Design System are you using?

    "carbon-components": "~10.17.0",
    "carbon-components-angular": "^4.13.0",

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

I am working in MCMP. This is expected to be fixed by OCTOBER.

## Steps to reproduce the issue

1. https://angular.carbondesignsystem.com/?path=/story/components-toggle--skeleton
2. Run Accessibility Checker 
3. You can see issue under Name, Role "Value Violations"

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/25789698/93881264-4ce11900-fcfc-11ea-9b23-097b11a4d106.png)

* Notes

**Comment by DusanMilko** [comment]
![Screen Shot 2020-09-22 at 11 35 32 AM](https://user-images.githubusercontent.com/302239/93911478-0f629880-fcc8-11ea-844c-6852c9cbe975.png)

toggles should have label text as color only is not accessible

**Comment by zvonimirfras** [comment]
Per Dusan's comment, this seems to be working as expected and users _should_ set on and off text for toggles in order to be accessible. Feel free to reopen with more info if needed.
